Description: Update code to work with newer version of github.com/pkg/errors
 Newer version is in debian. Submitted upstream and should be accepted.
Forwarded: https://github.com/git-lfs/git-lfs/issues/2390
Author: Stephen Gelman <ssgelm@gmail.com>
Last-Update: 2017-07-16
--- a/errors/types.go
+++ b/errors/types.go
@@ -336,8 +336,14 @@
 }
 
 func parentOf(err error) error {
-	if c, ok := err.(errorWithCause); ok {
-		return c.Cause()
+	type causer interface {
+		Cause() error
+	}
+
+	if c, ok := err.(causer); ok {
+		if innerC, innerOk := c.Cause().(causer); innerOk {
+			return innerC.Cause()
+		}
 	}
 
 	return nil
